src/__tests__/errors.test.tsx

```diff
@@ -31,19 +31,7 @@ test('errors emitted by the observable should be thrown during the react render
   // Note that the error is thrown later, during the render phase
   subject.next({error: true, message: 'Boom'})
 
-  const consoleErrorSpy = vitest.spyOn(globalThis.console, 'error').mockImplementation(() => {
-    // silence console.error()'s
-  })
   expect(() => rerender(<ObservableComponent />)).toThrowErrorMatchingInlineSnapshot(
     `[Error: Boom]`,
   )
-
-  // Assert that react warnings are captured and logged to the console
-  // note: this may change with React version
-  expect(consoleErrorSpy.mock.calls.flat().join('\n')).toMatchObject(
-    expect.stringContaining('Uncaught [Error: Boom]'),
-  )
-  expect(consoleErrorSpy.mock.calls.flat().join('\n')).toMatchObject(
-    expect.stringContaining('The above error occurred in the <ObservableComponent> component'),
-  )
```

Comment:
Turned out to be tricky to capture in react 19, it's too much of an implementation detail anyway as React might change how this error is rethrown at any time. It's enough that we test that the error is thrown during render.